dpe/src/commands/rotate_context.rs

```diff
@@ -33,6 +32,25 @@ impl RotateCtxCmd {
     const fn uses_target_is_default(&self) -> bool {
         self.flags.contains(RotateCtxFlags::TARGET_IS_DEFAULT)
     }
+
+    fn non_default_valid_handles_exist(
+        &self,
+        dpe: &mut DpeInstance,
+        locality: u32,
+        target_idx: usize,
+    ) -> bool {
+        dpe.contexts
+            .iter()
+            .enumerate()
+            .filter(|(idx, context)| {
```

Comment:
nit: You can use [any](https://doc.rust-lang.org/beta/core/iter/struct.Enumerate.html#method.any) instead of filter. Just a little more clear and stops after finding the first instance.

Reply:
Done